### PR TITLE
VUE-474 MG Campaign Homepage Driven By Contentful

### PR DESCRIPTION
**Pull request by eddieferrer**
* Preliminary work - Cleans up homepage template, create new homepage template  and story for MG campaign
* Content is being driven by contentful

STILL TODO:

- Add "form" element

### CODE REVIEW

File: src/pages/Homepage/Homepage.vue [hunk 1]
```diff
@@ -112,55 +107,32 @@ export default {
 		},
 		result({ data }) {
 			// Explicit lender preferences experiment - EXP-GROW-166-Aug2020
```
Comment (eddieferrer) [inline src/pages/Homepage/Homepage.vue:109]: Confirmed with Brandon that we can remove this experiment, it is 100% shown at this time.

File: src/pages/Homepage/MonthlyGoodHomepage.vue [hunk 2]
```diff
@@ -0,0 +1,491 @@
+<template>
```
Comment (eddieferrer) [inline src/pages/Homepage/MonthlyGoodHomepage.vue:1]: This template is just a copy of the current LendByCategoryHomepage for now, with the kiva classic buttons

**Review by mcstover — APPROVED** [review]
Great progress!

**Comment by eddieferrer** [comment]
Content being populated by contentful. 

Includes the work for: 

GROW-392, GROW-393, GROW-394, GROW-398, 

Screen Shot 2021-01-15 at 9 47 29 AM

File: src/pages/Homepage/MonthlyGoodHomepage.vue [hunk 3]
```diff
@@ -254,11 +299,71 @@ export default {
...
+		},
+		infoHeadline() {
+			const content = this.infoContentGroup?.contents?.find(contentItem => {
+				return contentItem.key === 'mg-homepage-info-text-header';
+			});
+			const text = content?.richText;
+			// contentful wraps all richText fields with a <p> tag,
+			// which makes them difficult to style as headers,
+			// this removes that wrapping tag
+			const options = {
```
Comment (mcstover) [inline src/pages/Homepage/MonthlyGoodHomepage.vue:348]: very interesting. Nice to see the ability to provide options though!

Comment (eddieferrer) [reply src/pages/Homepage/MonthlyGoodHomepage.vue:348]: We're using this in a couple of places, it should be extracted to contentful utils

**Review by mcstover — APPROVED** [review]

**Pull request merged**